tests/Functional/FunctionalTest.php

```diff
@@ -94,9 +122,10 @@ public function getFunctionalTests(): array
         $tests = [];
 
         foreach ($finder as $dir) {
-            $rstFilename = $dir->getPathname() . '/' . $dir->getFilename() . '.rst';
-            if (! file_exists($rstFilename)) {
-                throw new Exception(sprintf('Could not find functional test file "%s"', $rstFilename));
+            $rstFilename    = $dir->getPathname() . '/' . $dir->getFilename() . '.rst';
+            $indexFilename  = $dir->getPathname() . '/index.rst';
+            if (! file_exists($rstFilename) && ! file_exists($indexFilename)) {
```

Comment:
```suggestion
            if (! file_exists($rstFilename) || ! file_exists($indexFilename)) {
```

Reply:
just one of them two has to exist. Therefore and

Reply:
Sorry you're right, it's or in in the message because it's "Could not find functional test file (x Or y)" which translates to "Could not find functional test file x and could not find functional test file y", so this makes sense 👍